edificecms.com.website.test.json

```diff
@@ -0,0 +1,18 @@
+{
+  "providerId": "edificecms.com",
+  "providerName": "edificecms.com",
+  "serviceId": "website.test",
+  "serviceName": "edificecmsdns",
+  "version": 1,
+  "logoUrl": "https://beta.edificecms.com/builder/elements/images/uploads/project576/17089491464566.png",
+  "description": "Enables web hosting for edificecms domains.",
+  "variableDescription": "IP address for the domain A",
```

Comment:
I mean there are generally no variables so the description is not needed in fact.

Reply:
Variable description changed blank content